### PR TITLE
react assignment-1

### PR DESCRIPTION
**Pull request by nikhil1-successive**

### CODE REVIEW

File: assignment-1/src/Questions/Ques4/Counter.jsx [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+import ReactDOM from "react-dom/client";
+
+function Counter() {
+  const [val, setVal] = useState(0);
+
+  return (
+    <>
+     <h2 style={{color:"red",textDecoration:"underline"}}>Ques 4</h2>
+      <h1>Count:{val}</h1>
+      <button type="button" onClick={() => setVal(val+1)}>Inc</button>
```
Comment (shahbaz-successive) [inline assignment-1/src/Questions/Ques4/Counter.jsx:11]: Rename to Increment

File: assignment-1/src/Questions/Ques4/Counter.jsx [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+      &nbsp;&nbsp;
+      <button type="button" onClick={() => setVal(val-1)}>Dec</button>
```
Comment (shahbaz-successive) [inline assignment-1/src/Questions/Ques4/Counter.jsx:13]: Rename to Decrement

**Review by shahbaz-successive — CHANGES_REQUESTED** [review]
Feedback

File: assignment-1/src/App.js [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+    name:"doraemon",
+    email:"dora123@gmail.com",
+    avatarURL:"https://robohash.org/98b7768c88d1ef0833cc56b2be08cf2c?set=set4&bgset=&size=400x400"
+  }];
+  const temp=[10,20,40,-10,-20]
+  return (
+    <>
+      <Greeting />
+      <hr></hr>
+      <UserCard data={data}/> <hr></hr>
```
Comment (shahbaz-successive) [inline assignment-1/src/App.js:26]: Add one more instance of UserCard as asked in question

**Review by shahbaz-successive — CHANGES_REQUESTED** [review]
Feedback

File: assignment-1/src/Questions/Ques4/Counter.jsx (hunk 2 above)
Comment (nikhil1-successive) [reply assignment-1/src/Questions/Ques4/Counter.jsx]: Done

File: assignment-1/src/App.js (hunk 3 above)
Comment (nikhil1-successive) [reply assignment-1/src/App.js:26]: done

File: assignment-1/src/Questions/Ques4/Counter.jsx (hunk 1 above)
Comment (nikhil1-successive) [reply assignment-1/src/Questions/Ques4/Counter.jsx]: done

**Review by shahbaz-successive — APPROVED** [review]
👍

**Pull request merged**